### ISSUE TITLE
Updating SQL tables with indexes in sump_to_sql

### ISSUE THREAD

**Issue by pwalsh** [datahq/dataflows#170]
Relates to https://github.com/datahq/dataflows/pull/169

With support for index fields, the question arises regarding updating existing tables. There is no migration process now, so if one added indexes and wanted to apply them to an existing table when updating data, there is no possibility via the sql dumper.

Update runs could probably use Alembic and specifically https://alembic.sqlalchemy.org/en/latest/autogenerate.html to handle this.

**Comment by akariv** [comment]
Yes, using alembic would be pretty cool - perhaps as an improvement to the current tableschema-sql inner workings.
